src/safeds/data/tabular/plotting/_table_plotter.py

```diff
@@ -304,6 +314,8 @@ def line_plot(
             The name(s) of the column(s) to be plotted on the y-axis.
         show_confidence_interval:
             If the confidence interval is shown, per default True.
+        theme:
+            The color theme of the plot. Default is "light".
```

Comment:
The description for `theme` should be consistent for all functions. The description you used here seems best.